projects/go-lib/src/lib/components/go-modal/go-modal.component.spec.ts

```diff
@@ -100,18 +108,12 @@ describe('GoModalComponent', () => {
       expect(component.modalSize).toEqual('xl');
     });
 
-    it('handles when a modal size is set to unsupported size', () => {
-      component.currentComponent = new GoModalItem(GoTestModalHostComponent, { modalSize: 'abc' });
-
-      expect(component.modalSize).toEqual(component.defaultModalSize);
-
-      component.loadComponent();
-
-      expect(component.modalSize).toEqual(component.defaultModalSize);
-    });
-
```

Comment:
This test was removed because with the new type enforcements this can no longer be an issue.